src/Features.php

```diff
@@ -32,6 +32,7 @@ class Features {
 
 	public const PURGE_ON_FEATURE_CHANGES = 'purge_on_feature_changes';
 
+	public const GOOGLE_SHEET_REPLACES_SMARTSHEET = 'google_sheet_replaces_smartsheet';
```

Comment:
Since this has to be released early next week, why add a feature flag that we would have to manually switch for all the websites?

Reply:
How else would we switch and still provide continuity? :thinking: 

We need to release the settings field before people can enter their local sheets Google URL. This feature toggle allows to do that in a single release.

Also when we release it on Monday we still have a few days overlap where Smartsheet still works. So if an NRO would find a problem with their local sheet, they can switch it back to Smartsheet, fix it, and re-enable.

Reply:
Keep in mind that we also need to enter the local sheets IDs in each site anyway.

Reply:
Ah right, I was thinking mostly about the Global Sheet. But for local projects, NROs should gradually update and switch :+1: